lib/Hakyll/Core/Rules.hs

```diff
@@ -130,18 +143,100 @@ matchInternal pattern getIDs rules = do
     setMatches ids env = env {rulesMatches = ids}
 
 --------------------------------------------------------------------------------
-match :: Pattern -> Rules () -> Rules ()
+{- | Add a selection of which input files to process (using the given
+[glob pattern](https://w.wiki/4Wsx)) to the given remaining 'Rules' pipeline.
```

Comment:
I can't get this link to render correctly in the docs. Any ideas?

Reply:
Haddock has its own markup for links: https://www.haskell.org/haddock/doc/html/ch03s08.html#idm140354810727840

Also, are there any reason to use a URL shortener here? Why not link to Wikipedia directly?

Reply:
Markdown style links are supported https://haskell-haddock.readthedocs.io/en/latest/markup.html#links (the docs on haskell.org seem to be terribly outdated). I got it working.

I used a shortener to test if the parenthesis in the url `https://en.wikipedia.org/wiki/Glob_(programming)` are a problem; they are, but can be escaped. 

Apparently, there's another bug/quirk in Haddock:
> {- | Add a selection...
> given [glob pattern](https://en.wikipedia.org/wiki/Glob_(programming\))) ...

renders correctly, but having such a link at the beginning of a block comment line doesn't:
> {- | Add a selection ...
> [glob pattern](https://en.wikipedia.org/wiki/Glob_(programming\))) to the given remaining 'Rules' pipeline.

I'll add a bug report on Haddock.